src/ORM/Behavior/Translate/ShadowTableStrategy.php

```diff
@@ -135,7 +135,10 @@ public function beforeFind(EventInterface $event, Query $query, ArrayObject $opt
 
         $fieldsAdded = $this->addFieldsToQuery($query, $config);
         $orderByTranslatedField = $this->iterateClause($query, 'order', $config);
-        $filteredByTranslatedField = $this->traverseClause($query, 'where', $config);
+        $filteredByTranslatedField =
+            $this->traverseClause($query, 'where', $config) ||
+            $config['onlyTranslated'] ||
+            ($options['filterByCurrentLocale'] ?? null);
```

Comment:
Is this a new option added?

Reply:
`filterByCurrentLocale`? No, it exists ever since 3.0, but it seems that it was never properly documented.

Reply:
Ah, I see now that it's an existing option, albeit undocumented.